std_msgs/Header header
string[] family_names
Center[] centers
bool detected
int64 total_tags
